packages/client/kintsugi/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM node:14-alpine as build
+FROM node:16-alpine as build
 WORKDIR /usr/app
-RUN apk update && apk add --no-cache g++ make python bash git && rm -rf /var/cache/apk/*
+RUN apk update && apk add --no-cache bash git && rm -rf /var/cache/apk/*
 
 WORKDIR /usr/app
```

Comment:
Isn't this duplicative?  I'm still a novice at Docker but doesn't this create a new layer in the image unnecessarily?

Reply:
isn't what duplicative? here i just simplified some of the modules that were being installed (didn't need g++, make, python)

Reply:
Oh, sorry, it commented on the wrong line.  I meant line 5
`WORKDIR /usr/app`

Isn't this duplicative of line 2?

Reply:
ah hmm, `WORKDIR` is just like `cd`, so yeah looks duplicative but probably harmless, i can try without it when i give the docker image a spin today and can remove if unnecessary